modules/oe_webtools_media/src/Plugin/media/Source/Webtools.php

```diff
@@ -154,7 +154,7 @@ public function createSourceField(MediaTypeInterface $type) {
 
     $generator_link = Link::fromTextAndUrl(
       $this->t('Webtools wizard'),
-      Url::fromUri('https://europa.eu/webtools/mgmt/wizard/')
+      Url::fromUri('https://webtools.europa.eu/mgmt/wizard/')
```

Comment:
The URL of the Wizard is now https://webtools.europa.eu/tools/#/wizards

Reply:
This seems relevant.